clidocstool.go

```diff
@@ -18,6 +18,10 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const (
+	AnnotationExternalUrl = "docs.external.url"
```

Comment:
Could use a short comment to describe what it's for


(Also thinking if we should have a list of common annotations that we use in our CLIs 🤔)

Reply:
If you think of others annotations we can add them in a follow-up yes